format/Message.fbs

```diff
@@ -109,12 +134,16 @@ table Field {
   dictionary: long;
   // children apply only to Nested data types like Struct, List and Union
   children: [Field];
+  /// layout of buffers produced for this type (as derived from the Type)
+  /// does not include children
```

Comment:
What does "does not include children" mean? I would expect this to list all buffers for a batch.